src/HumiditySensorDevice.ts

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env node
```

Comment:
Can you move those files to an examples folder?
I think this will be the best place for them.

Reply:
I agree - Alternative would be to enhance the "Device.ts" to allow to specify a -type parameter and then have just small stub files with the "which lucster to add and how to configure"??